rust/cubestore/cubestore/src/bin/cubestored.rs

```diff
@@ -36,12 +36,7 @@ fn main() {
     let metrics_port = std::env::var("CUBESTORE_METRICS_PORT").unwrap_or("8125".to_string());
     let metrics_server_address = format!("{}:{}", metrics_addr, metrics_port);
 
-    init_metrics(
-        "127.0.0.1:0",
-        metrics_server_address,
-        metrics_format,
-        vec![],
-    );
+    init_metrics("0.0.0.0:0", metrics_server_address, metrics_format, vec![]);
```

Comment:
@FlipEnergy I realized that in some cases exposing metrics via all [public] interfaces might have a security impact. So it would be much better to have this as a controlled option. Let's use an env with the default value of `127.0.0.1` with this. So those who need it (like you) may set the env to the desired value without impacting anyone else.